### PR TITLE
[docs] update the documents

### PR DESCRIPTION
**Pull request by wsstriving**
Add documents for supervised/self-supervised learning and .etc.

### CODE REVIEW

**Comment by wsstriving** [comment]
@cdliang11 Chengdong, can you help to check how to put this online similiar to https://wenet.org.cn/wenet/reference.html?

File: docs/python_package.md
Comment (wsstriving) [inline docs/python_package.md:1]: pycharm reformat

File: docs/speaker_recognition_papers.md
Comment (wsstriving) [inline docs/speaker_recognition_papers.md:1]: Unified the title format

**Comment by cdliang11** [comment]
> @cdliang11 Chengdong, can you help to check how to put this online similiar to https://wenet.org.cn/wenet/reference.html?

Okay, I will support it in future PR. The current PR can be merged.

**Review by cdliang11 — APPROVED** [review]

**Pull request merged**